content/en/docs/concepts/storage/volumes.md

```diff
@@ -1263,47 +1235,42 @@ spec:
 
 ## Resources
 
-The storage media (Disk, SSD, etc.) of an `emptyDir` volume is determined by the
+The storage media (such as Disk or SSD) of an `emptyDir` volume is determined by the
 medium of the filesystem holding the kubelet root dir (typically
-`/var/lib/kubelet`).  There is no limit on how much space an `emptyDir` or
-`hostPath` volume can consume, and no isolation between Containers or between
-Pods.
+`/var/lib/kubelet`). There is no limit on how much space an `emptyDir` or
+`hostPath` volume can consume, and no isolation between containers or between
+pods.
```

Comment:
Aside: this is slightly out of date / inaccurate.
1. You can have memory-backed `emptyDir` volumes.
1. Calling `/var/lib/kubelet` a root directory of any kind is a bit misleading.

Reply:
Earlier, the page mentions that you can set the `emptyDir.medium` field to `"Memory"` to get memory-backed ephemeral local storage.

Reply:
Yes, I read through this and found that the content is confusing (or out of place).